Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -461,6 +464,9 @@ class BrowserViewController: UIViewController {
 
         // Awesomebar Location Telemetry
         SearchBarSettingsViewModel.recordLocationTelemetry(for: isBottomSearchBar ? .bottom : .top)
+        if shouldUseOverlayManager {
+            overlayManager = DefaultOverlayModeManager(urlBarView: urlBar)
+        }
```

Comment:
nit, just adding a space otherwise it seems the overlay manager is part of the telemetry comment
```suggestion
        // Awesomebar Location Telemetry
        SearchBarSettingsViewModel.recordLocationTelemetry(for: isBottomSearchBar ? .bottom : .top)

        if shouldUseOverlayManager {
            overlayManager = DefaultOverlayModeManager(urlBarView: urlBar)
        }
```